js/script.js

```diff
@@ -86,42 +87,57 @@
 			memTotalGB = memTotal / (1024 * 1024),
 			memUsageGB = (memTotal - memFree) / (1024 * 1024);
 
+		var swapTotalBytes = swapTotal * 1024,
+			swapUsageBytes = (swapTotal - swapFree) * 1024,
+			swapTotalGB = swapTotal / (1024 * 1024),
+			swapUsageGB = (swapTotal - swapFree) / (1024 * 1024);
+
+	  if (memTotalGB > swapTotalGB) {
+			var maxValueOfChart = memTotalGB;
+		} else {
+			var maxValueOfChart = swapTotalGB;
+		}
+
 		if (typeof memoryUsageChart === 'undefined') {
 			memoryUsageChart = new SmoothieChart(
 			{
-				millisPerPixel:250, 
-				maxValue:memTotalGB, 
-				minValue:0, 
+				millisPerPixel:250,
+				maxValue:maxValueOfChart,
+				minValue:0,
 				grid:{fillStyle:'rgba(0,0,0,0.03)',strokeStyle:'transparent'},
 				labels:{fillStyle:'rgba(0,0,0,0.4)', fontSize:12}
 			});
 			memoryUsageChart.streamTo(document.getElementById("memorycanvas"), 1000/*delay*/);
 			memoryUsageLine = new TimeSeries();
 			memoryUsageChart.addTimeSeries(memoryUsageLine, {lineWidth:1, strokeStyle:'rgb(0, 255, 0)', fillStyle:'rgba(0, 255, 0, 0.2)'});
+			swapUsageLine = new TimeSeries();
+      memoryUsageChart.addTimeSeries(swapUsageLine, {lineWidth:1, strokeStyle:'rgb(255, 0, 0)', fillStyle:'rgba(255, 0, 0, 0.2)'});
 		}
 
 		$('#memFooterInfo').text(t('serverinfo', 'Total')+": "+OC.Util.humanFileSize(memTotalBytes)+" - "+t('serverinfo', 'Current usage')+": "+OC.Util.humanFileSize(memUsageBytes));
 		memoryUsageLine.append(new Date().getTime(), memUsageGB);
+		$('#swapFooterInfo').text("Total (SWAP): "+OC.Util.humanFileSize(swapTotalBytes)+" - Current usage: "+OC.Util.humanFileSize(swapUsageBytes));
```

Comment:
Add "Current usage" into `t()` JS-function to translate the string like two lines above at `#memFooterInfo`. Further I'd suggest format like:
"RAM: Total: X GB - Current usage: X GB
SWAP: Total X GB - Current usage: X GB"